Source/PropertyTools.Wpf/PropertyGrid/PropertyGrid.cs

```diff
@@ -1015,15 +1015,14 @@ public virtual void CreateControls(object instance, IEnumerable<Tab> tabs)
 
                 var tabItem = new TabItem { Header = tab, Padding = new Thickness(4), Name = tab.Id ?? string.Empty };
 
-                if (instance is IDataErrorInfo dataErrorInfoInstance)
+                if (instance is INotifyDataErrorInfo notifyDataErrorInfoInstance)

```

Comment:
second thought: I think all this functionality could be delegate to the control factory. e.g. `this.ControlFactory.UpdateTab(tab, instance)`. Can we include that change here?

Reply:
I think this will make the code both easier to read and more extensible